common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -204,11 +211,14 @@
         ;; Static HTML resources, typically API documentation which needs endpoint URLs replaced
         (GET ["/:page", :page #".*\.html$"] {headers :headers, {page :page} :params}
           (when-let [resource (site-resource page)]
-            (let [cmr-root (str public-protocol "://" (headers "host") relative-root-url)]
+            (let [cmr-root (str public-protocol "://" (headers "host") relative-root-url)
+                  site-example-provider (get site-provider-map (headers "host") "PROV1") 
+                  cmr-example-collection-id (str "C1234567-" site-example-provider)]
```

Comment:
I was really hoping we could provide a real collection concept ID for each environment so that for the most part they could execute the example query and get back results. It seems like we should be able to read from an environment variable for the collection ID.

Could we add
`(defconfig example-collection-id ...)`, or do you see an issue with that?

Reply:
The issue is we have many dummy values and ids in the existing example queries. Only changing the new constraint to use the real values is not going to get the results back. Also, with defconfig, someone at the sites has to set it to a real value...    Communicated this to Chris. He said it's no big deal.  Since we can't make the whole collection concept id real, I could just use a dummy one instead of using the dummy C1234567- with the real provider name. But I will just leave it as is. In case we want to change all the values to real values in the examples.